layouts/_default/list.html

```
@@ -1,3 +1,5 @@
+{{ $paginator := .Paginator 5 }}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)